packages/discord.js-utilities/src/lib/PaginatedMessages/PaginatedFieldMessageEmbed.ts

```diff
@@ -0,0 +1,167 @@
+import { MessageEmbed, MessageEmbedOptions } from 'discord.js';
+import { PaginatedMessage } from './PaginatedMessage';
+
+/**
+ * This is a utility of {@link PaginatedMessage}, but paginating only the field of an embed.
+ * You must either use this class directly or extend it.
+ *
+ * @example
+ * ```typescript
+ * import { PaginationFieldMessageEmbed } from '@sapphire/discord.js-utilities';
+ *
+ * new PaginationFieldMessageEmbed()
+ *    .setTitleField('Test pager field')
+ *    .setTemplate({ embed })
+ *    .setItems([
+ *       { title: 'Sapphire Framework', value: 'discord.js Framework' },
+ *       { title: 'Sapphire Framework 2', value: 'discord.js Framework 2' },
+ *       { title: 'Sapphire Framework 3', value: 'Discord.js Framework 3' }
+ *     ])
+ *    .formatField((item) => `${item.title}\n${item.value}`)
+ *    .setItemsPerPage(2)
+ *    .make()
+ *    .run(message);
+ * ```
+ */
+export class PaginationFieldMessageEmbed<T> extends PaginatedMessage {
+	private embedTemplate: MessageEmbed = new MessageEmbed();
+	private totalPages = 0;
+	private items: T[] = [];
+	private itemsPerPage = 10;
+	private fieldTitle = '';
+
+	/**
+	 * Set the items to paginate.
+	 * @param items The pages to set
+	 */
+	public setItems(items: T[]) {
+		this.items = items;
+		return this;
+	}
+
+	/**
+	 * Set the title of the embed field that will be used to paginate the items.
+	 * @param title Field title
+	 */
+	public setTitleField(title: string) {
+		this.fieldTitle = title;
+		return this;
+	}
+
+	/**
+	 * Sets the amount of items that should be shown per page.
+	 * @param itemsPerPage The number of items
+	 */
+	public setItemsPerPage(itemsPerPage: number) {
+		this.itemsPerPage = itemsPerPage;
+		return this;
+	}
+
+	/**
+	 * Template to be used to display the pagination of the articles. This template can be configured by the `new MessageEmbed()` class or directly by means of an object with embed options.
```

Comment:
```suggestion
	 * Sets the template to be used to display the embed fields as pages. This template can either be set from a template {@link MessageEmbed} instance or an object with embed options.
```
I'm not sure if typedoc can link or reference external sources, but if not, replace `{@link MessageEmbed}` here with `MessageEmbed`.